core-kotlin-modules/core-kotlin-datastructures/src/test/kotlin/com/baeldung/frequencymap/FrequencyMapUnitTest.kt

```diff
@@ -0,0 +1,123 @@
+package com.baeldung.frequencymap
+
+import org.junit.Test
+import java.util.*
+import kotlin.test.assertContentEquals
+import kotlin.test.assertEquals
+
+class FrequencyMapUnitTest {
+
+    @Test
+    fun `test frequency map using mutable map method`() {
+        val list = listOf(1, 2, 1, 3, 2, 4, 4, 7,9,7,3,2,1)
```

Comment:
```suggestion
        val list = listOf(1, 2, 1, 3, 2, 4, 4, 7, 9, 7, 3, 2, 1)
```

Reply:
We need to add spaces here for readability. Let's apply this change in the other tests, too.